### ISSUE TITLE
Adding text to note breaks formatting

### ISSUE THREAD

**Issue by loek-solutionslab** [raycast/extensions#14915]
### Extension

https://www.raycast.com/raycast/apple-notes

### Raycast Version

Version 1.84.2

### macOS Version

macOS Sequoia 15.0.1

### Description

Adding text to a note breaks the layout of the note. It deleted my checkboxes and H1,h2 etc. formatting.

### Steps To Reproduce

create a note in the latest note app, add some formatting like H2 headers and checkboxes, add text to note via the raycast command → watch the formatting disapear.

### Current Behaviour

.

### Expected Behaviour

.

**Comment by thomaslombart** [comment]
Indeed, that's frustrating. Unfortunately, there's no other way to achieve that as we're quite limited with Apple's APIs. Maybe it'd be better to kill the feature since it can does more harm than good.

**Comment by loek-solutionslab** [comment]
Yeah, it’s too bad
